Subject: [PATCH] tools:cesa: Set CC and LD flags

Recipe mv-cesa-tool uses mudule bbclass that unset CC and LD, but
mv_cesa_tool.c needs to use STAGING_DIR_TARGET path to include C headers.

Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>
---
 tools/cesa/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/cesa/Makefile b/tools/cesa/Makefile
index 5be03de..c8804ce 100644
--- a/tools/cesa/Makefile
+++ b/tools/cesa/Makefile
@@ -6,8 +6,8 @@ ifndef KSRC
 KSRC  := $(srctree)/../..
 endif
 
-CC = $(CROSS_COMPILE)gcc
-LD = $(CROSS_COMPILE)ld
+CC = $(CROSS_COMPILE)gcc ${HOST_CC_ARCH} ${TOOLCHAIN_OPTIONS}
+LD = $(CROSS_COMPILE)ld ${HOST_CC_ARCH} ${TOOLCHAIN_OPTIONS}
 CFLAGS = -Wall -Wextra
 CFLAGS += -I$(KSRC)/drivers/crypto/mvebu_cesa/
 
-- 
2.1.4

